firebase/firebaseConfig.ts

```diff
@@ -14,5 +16,7 @@ const firebaseConfig = {
 // Initialize Firebase
 export const FIREBASE_APP = initializeApp(firebaseConfig) 
 export const FIREBASE_AUTH = getAuth(FIREBASE_APP) 
+export const FIRESTORE_DB = getFirestore(FIREBASE_APP)
```

Comment:
I suggest a shorter name for this variable, something like `db`, since it will be used many times in many different files